contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -2957,6 +2957,49 @@ SELECT
 WHERE FALSE; -- This condition will ensure that the view is empty
 GRANT SELECT ON sys.numbered_procedures TO PUBLIC;
 
+CREATE OR REPLACE FUNCTION sys.fn_listextendedproperty (
```

Comment:
please add upgrade tests to validate calling this function from a view/function/procedure across release works

Reply:
Added the tests